toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -71,133 +152,236 @@ func CustomizeImageWithConfigFile(buildDir string, configFile string, imageFile
 	return nil
 }
 
+func cleanUp(ic *ImageCustomizerParameters) error {
+
+	var allErrors error
```

Comment:
Why is this called `allErrors`?

Reply:
This function used to clean-up multiple resources and would not return after the first error to attempt to do further clean-up. Right now, this is unnecessary - so, I'll revert to simple err check/return pattern.
pending.